### DISCUSSION TITLE
Can you keep alDente running ONLY in Menubar?

### DISCUSSION THREAD

**Discussion by fffelan1** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#886] [Q&A]
Hi, 

On my MacBook Pro m1 the App completely looses its functionality after quitting the app with cmd + q. Other Menubar Apps keep running in the menubar even when you close the application. 

Is this behavior intended? Will the pro version allow me to do this?

Thanks,
Phelan

**Comment by MatthiasKerbl** [comment]
Hi @fffelan1,

As far as I know, cmd+q always closes apps completely.
If you want to have AlDente running in the background, even without the menubar item, you need AlDente Pro which has a setting called "Hide Menubar Icon".

Cheers,
Matthias

**Reply by fffelan1** [reply]
Thanks for the quick response,

No, I want the app to vanish from the dock and the cmd + tab app switcher, but remain in the menubar.


<img width="1512" alt="Bildschirmfoto 2023-01-25 um 13 55 13" src="https://user-images.githubusercontent.com/70589900/214568955-ae373d33-a7a8-41a8-ab60-70f539269c34.png">

**Reply by MatthiasKerbl** [reply]
Then you need to go to Settings/Appearance and disable "Show Dock Icon".

**Comment by fffelan1** [comment]
aaaah. 
I overlooked that setting. thank you! problem solved.

**Comment by matchavez** [comment]
I overlooked this too until I found it. The reason is that the window was extended below the bottom of my screen.
